.github/workflows/release-windows-vcpkg-cmake.yml

```diff
@@ -154,16 +154,18 @@ jobs:
           Based on branch: ${{github.ref_name}}
 
           #### Changes
+          "
           
+          Add-Content -Path ./change_note.txt -Value @'
           ${{steps.build_changelog.outputs.changelog}}
-          "          
+          '@
 
           $tagExist = gh api --silent "repos/:owner/:repo/git/refs/tags/${{github.ref_name}}"
```

Comment:
This seems not correct.

${{github.ref_name}} should be $tagName

Reply:
You are right